### ISSUE TITLE
git-quick-stats should ignore format.pretty in gitconfig

### ISSUE THREAD

**Issue by aaronjensen** [arzzen/git-quick-stats#19]
As mentioned in #16,

Apparently git-quick-stats is using my overridden pretty format from my git config. To repro, add this to your gitconfig:

```
[format]
  pretty=format:%C(yellow)%h%Creset -%C(red)%d%Creset %s %Cgreen(%ar) %C(bold blue)<%an>%Creset
```

It reports no contributors for the contribution stats.

You may need to hard code the format you use, or maybe temporarily unset HOME so it ignores the gitconfig.

**Comment by arzzen** [comment]
fixed in ecdb11c

**Comment by aaronjensen** [comment]
Works great, thanks!
